src/features/TodoItem.js

```diff
@@ -1,5 +1,29 @@
-import React from "react";
+import React, { useState } from "react";
+import { useDispatch } from "react-redux/es/exports";
+import { changeToggle, deleteItem } from "./todoSlice";
+import "./display.css"
 
 export default function TodoItem(props) {
-  return <div>{props.todo}</div>;
+  const [toggle, setToggle] = useState(true);
+  const dispatch = useDispatch();
+  const handleClick = (event) => {
+    setToggle(!toggle);
+    console.log(toggle);
+    if (!toggle) {
+      event.target.style.removeProperty("text-decoration");
+    } else {
+      event.target.style.setProperty("text-decoration", "line-through");
+    }
+    dispatch(changeToggle(props.todo.id));
```

Comment:
should use css and conditionally depend on the "done" state

example provided
className={done ? "done" : ""}
.done {
    text-decoration: line-through;
}